---
layout: article
title: "Panda Project"
doi: XXXXXX
date: 2015-09-14
authors:
  - name: Yi Zhang
    email: yizhang@ablife.cc
    url: https://github.com/
    affiliations:
      - address: CEO & Chief Scientist ABLife, Inc. Optics Valley International Biomedical Park, Building 9-4,  
        url: http://ablife.cc/
    funders:
      - name: 
        url:
        doi:
        awards: [ ABL2013-05004 ]
  - name: Jie Huang
    email: jiehuang@ablife.cc
    affiliations:
      - address: Optics Valley International Biomedical Park, Building 9-4, Wuhan,Chnia
        url: https://github.com/
abstract: |
        **Please note: This is an experimental [Paper Now](https://github.com/PeerJ/paper-now) version of [this PeerJ article](https://peerj.com/articles/858/). No guarantees are given for the correctness or completeness of this experimental version.**

        The giant panda *(Ailuropoda melanoleura)*, one of the most endangered mammals with only ~2000 adult survived and less 10 new-born captive children each year in the southwestern China, is widely interested by its carnivore-like gastrointestine but herbivore-like diet that gives them a high risk of extinction (1, 2). For this mammal, a short and relatively simple digestive tract of as typical carnivore without a rumen for fermentation makes this mammal processing bamboo with a low digestibility (only ~17%), and thus they have to consuming a lot of bamboo (~12.5 kg) and time (up to 14 h) to ensure the least nutrition each day (2-4). Recently, the advent of high-throughput sequencing technologies has led to a turning point in our understanding of this animal intestinal microbial colonization. Several researches revealed the constructions and part potential functions of gut flora for this mammal by metagenomics procedure (5-8), which were more comprehensive and accurate than early culture-dependent studies (9, 10) or traditional molecular approaches (11, 12). However, with increasing projects we found there are large community variations between the different researches below, for example, Li Y, et al. (5) identified gut microbiota were contrary to Tun HM, et al. (8) with much lower diversity and dominated mainly by class *Streptococcus* but not *Gammaproteobacteria*.

keywords: Nutritional adaptation,  Giant panda, Metagenomics,  Gut microbiota

acknowledgements: "We thank XXXX for sample preparation, illumina device operations, and insightful discussions, as well as XXXX for helpful ideas and suggestions. Our gratitude also belongs to two anonymous reviewers whose valuable suggestions improved this manuscript greatly."
---
